clang/docs/ReleaseNotes.rst

```diff
@@ -335,6 +336,22 @@ Attribute Changes in Clang
 - Clang now warns that the ``exclude_from_explicit_instantiation`` attribute
   is ignored when applied to a local class or a member thereof.
 
+- The ``counted_by`` attribute can now be late parsed in C when
```

Comment:
I can mention it but I'm wondering if it should be in the release notes at all because this patch only makes parsing work. I've not done anything on the codegen side to make use of the attribute on pointers.

Reply:
If it doesn't have a use yet, maybe you don't need to add it in the release notes?

Reply:
So to add some context here. The original reason there was a release note is because in #88596 (which this patch builds upon) @AaronBallman asked for there to be a release note about the new `-fexperimental-late-parse-attributes` flag. This patch means `-fexperimental-late-parse-attributes` actually does something now (before it did nothing) so I updated the release notes on that flag and it "felt" like it made sense to explain in the release notes the change on the attributes in relation to the flag.

I don't have strong opinions here. I'm ok to drop the changes to the release notes or keep them or something in between. @AaronBallman any opinions here?

Reply:
I don't have a strong opinion either. :-) It's fine to keep the blurb in.